vnext/ReactUWP/Modules/DeviceInfoModule.cpp

```diff
@@ -15,7 +15,29 @@ namespace uwp {
 //
 DeviceInfo::DeviceInfo(const std::shared_ptr<IReactInstance> &reactInstance)
     : m_wkReactInstance(reactInstance) {
-  update();
+  try {
+    update();
+  } catch (winrt::hresult_error const &ex) {
+    if (ex.code() != 0x80070490) {
+      // PEER_E_NOT_FOUND Expected when app starts in the background.
+      winrt::throw_last_error();
+    }
+    // Set fake data when we are in the background.
+    // Using 1 for all values since it should be obviously fake, and avoids
+    // consumers dividing by 0
+    m_dimensions = getFollyDimensions(1, 1, 1, 1, 1, 1, 1);
+    m_leavingBackgroundRevoker =
+        winrt::Windows::UI::Xaml::Application::Current().LeavingBackground(
+            winrt::auto_revoke,
+            [this, m_wkReactInstance = m_wkReactInstance](
```

Comment:
>m_wkReactInstance [](start = 19, length = 17)

why do you add m_wkReactInstance here, but you didn't add m_leavingBackgroundRevoker?